sdk/loadtestservice/loadtestservice/package.json

```
@@ -0,0 +1,115 @@
+{
+  "name": "@azure-rest/loadtestservice",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/loadtestservice.d.ts",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/undefined/README.md",
```

@joheredi is this a gap in the generator?
```suggestion
  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/loadtestservice/loadtestservice/README.md",
```

---

It may be @qiaozha, can you please take a look at this?

---

It's corrected now as I regenerated the code.